src/resmom/mom_server.c

```diff
@@ -1102,6 +1102,8 @@ is_request(int stream, int version)
 							log_err(errno, __func__, MALLOC_ERR_MSG);
 							goto err;
 					}
+					memset(phook_output->reject_errcode, 0, sizeof(int));
+
```

Comment:
This might be more than needed. How about... phook_output->reject_errcode = 0;

Reply:
Thank you Mike, modified the statement.